backend/app.py

```diff
@@ -1,22 +1,29 @@
 """TermHub backend
-https://github.com/tiangolo/fastapi"""
+
+Resources
```

Comment:
FYI: Added list of resources we probably want to remember / look up to the top of file.